wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/EnhancedDraft3Ast.scala

```diff
@@ -44,7 +44,7 @@ object EnhancedDraft3Ast {
     }
 
     def getAttributeAsVectors[A, B](attr: String, astName1: String, astName2: String)
-                                   (implicit astNodeToA: CheckedAtoB[AstNode, A], astNodeToB:CheckedAtoB[AstNode, B]
+                                   (implicit astNodeToA: CheckedAtoB[AstNode, A], astNodeToB: CheckedAtoB[AstNode, B]
```

Comment:
I was thinking about this function. If you feel up to it, I'd suggest you make this change - but maybe as a subsequent PR if you want to merge this one:

 - We should have a super type for everything we might want to be in the "body" attribute (eg `WorkflowBodyElement`)
 - Every case class for a body element extends it, eg `case class WorkflowOutputsSection(...) extends WorkflowBodyElement`
 - We can have a top level `CheckedAtoB[Ast, WorkflowBodyElement]` which delegates to (eg) `CheckedAtoB[Ast, WorkflowOutputsElement]` based on the name of the Ast it receives.
- Then in the `CheckedAtoB[Ast, WorkflowDefinitionElement]` function we'll end up with a set of `WorkflowBodyElements`. We'll need to do the filters there to get appropriate vectors for the `WorkflowDefinitionElement` fields.

Reply:
Sounds good to me. I think that I will wait and add that as part of my next PR, since I've already begun working on some other changes. Do you think a super type of CallableBodyElement would be more useful?

Reply:
Probably best to keep workflow and task body elements as separate supertype `trait`s - the set of things that can go in a workflow is different from the set of things that can go in a task.

Although I imagine `case class OutputsSection` will have to both, ie `extends WorkflowBodyElement with TaskBodyElement`

Reply:
Traits are pretty cool!